### DISCUSSION TITLE
南极谋杀案（十三）- 拉格朗日罢工 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#451] [General]
# 南极谋杀案（十三）- 拉格朗日罢工 - Miao Yu | 于淼

https://yufree.cn/cn/2024/01/06/antarictic-murder/

**Comment by earfanfan** [comment]
三水，你好：
  为撒你写完拉格朗日罢工以后，自己也罢工了，哈哈，撒时候爬回来耕博客呢？
